secondarylib-feedback/src/main/scala/com/socrata/datacoordinator/secondary/feedback/FeedbackCookie.scala

```diff
@@ -24,12 +39,16 @@ object FeedbackCookie {
     Some(CompactJsonWriter.toString(feedbackCookieCodec.encode(fbc)))
   }
 
-  def decode(ck: Cookie): Option[FeedbackCookie] = ck match {
+  def decode(ck: Cookie): Option[FeedbackCookie] = try {
+    ck match {
       case Some(str) => JsonUtil.parseJson[FeedbackCookie](str) match {
         case Right(result) => Some(result)
         case Left(error) => None // safely handle cookie corruption
       }
       case None => None
+    }
+  } catch {
+    case _ : JsonUnknownIdentifier => None // safely handle cookie corruption
```

Comment:
ParseJson can get some error not wrapped in left error?  I wonder you can use JsonUtil.parseJson[FeedbackCookie]("str").right.toOption

Reply:
Yeah, I was kind of surprised that it could throw an exception.... but it was a bug I fixed

Reply:
OK, you still can switch to the more compact version in the first half.

Reply:
JsonUnknownIdentifier is not the only exception that can be thrown -- but all exceptions that are are subclasses of JsonReaderException.